data/web/lang/lang.en.php

```diff
@@ -341,6 +341,8 @@
 $lang['edit']['backup_mx_options'] = 'Backup MX options';
 $lang['edit']['relay_domain'] = 'Relay domain';
 $lang['edit']['relay_all'] = 'Relay all recipients';
+$lang['edit']['relay_sender'] = 'Relay all sending mails to the Relayhost';
+$lang['edit']['relay_recipient'] = 'Relay all reciving mails to the Relayhost';
```

Comment:
Should be "receiving" instead of "reciving"

Reply:
Arg, I hate typos ;-)